webapp/channels/src/components/threading/global_threads/thread_item/index.ts

```diff
@@ -42,7 +40,4 @@ function makeMapStateToProps() {
     };
 }
 
-export default compose(
-    connect(makeMapStateToProps),
-    memo,
```

Comment:
We are already calling memo in the component file. There is no need for double memo.